packages/core/src/browser/style/tree.css

```diff
@@ -79,7 +79,7 @@
     font-family: FontAwesome;
     font-size: calc(var(--theia-content-font-size) * 0.8);
     content: "\f0da";
-    transform: rotate(45deg);
+    transform: rotate(90deg);
```

Comment:
@kaiyue0329 do you mind explaining why this change is necessary?

Reply:
I changed it so that the arrow can point to the vertical line when a folder is expanded in the tree.